master/static/srv/entrypoint.sh

```diff
@@ -4,28 +4,9 @@ source /run/determined/task-setup.sh
 
 set -e
 
-STARTUP_HOOK="startup-hook.sh"
-export PATH="/run/determined/pythonuserbase/bin:$PATH"
-
-# If HOME is not explicitly set for a container, libcontainer (Docker) will
-# try to guess it by reading /etc/password directly, which will not work with
-# our libnss_determined plugin (or any user-defined NSS plugin in a container).
-# The default is "/", but HOME must be a writable location for distributed
-# training, so we try to query the user system for a valid HOME, or default to
-# the working directory otherwise.
-if [ "$HOME" = "/" ]; then
```

Comment:
for posterity, why exactly did we need to remove do this?

Reply:
it moved to task-setup.sh